snuba/datasets/storages/metrics.py

```diff
@@ -143,6 +143,18 @@ def ignore_policy_closure() -> DeadLetterQueuePolicy:
     write_format=WriteFormat.VALUES,
 )
 
+org_counters_storage = ReadableTableStorage(
+    storage_key=StorageKey.METRICS_COUNTERS,
```

Comment:
You cannot reuse the same storage key for two different storages. Please create a new one.